client/boot/index.js

```diff
@@ -239,6 +239,11 @@ function reduxStoreReady( reduxStore ) {
 				user_id: user.get().ID,
 				calypso_env: config( 'env_id' )
 			} );
+
+			if ( global.window && window.app && window.app.commitChecksum ) {
```

Comment:
Isn't a typeof check safer here? global.window will throw a reference error on the client.